src/test/java/org/gridsuite/modification/server/modifications/tabularmodifications/TabularGeneratorModificationsTest.java

```diff
@@ -87,48 +88,50 @@ protected void assertAfterNetworkModificationDeletion() {
-                .modifications(modifications)
-                .build();
-        UUID modificationUuid = saveModification(modificationInfos);
+    public void testSqlRequestsCountOnGetModification() throws Exception {
+        UUID tabularWith1ModificationUuid = createTabularGeneratorModification(1);
         reset();
-
-        mockMvc.perform(get("/v1/network-modifications/{uuid}", modificationUuid)).andExpectAll(
-                        status().isOk(), content().contentType(MediaType.APPLICATION_JSON))
-                .andReturn();
+        ApiUtils.getModification(mockMvc, tabularWith1ModificationUuid); // Getting one tabular modification with one sub-modification
         assertSelectCount(3);
 
-        modifications = List.of(
-                GeneratorModificationInfos.builder().equipmentId("idGenerator").maxActivePower(new AttributeModification<>(300., OperationType.SET)).build(),
-                GeneratorModificationInfos.builder().equipmentId("v5generator").maxActivePower(new AttributeModification<>(300., OperationType.SET)).build(),
-                GeneratorModificationInfos.builder().equipmentId("v6generator").maxActivePower(new AttributeModification<>(300., OperationType.SET)).build()
-        );
-        modificationInfos = TabularModificationInfos.builder()
-                .modificationType(ModificationType.GENERATOR_MODIFICATION)
-                .modifications(modifications)
-                .build();
-        modificationUuid = saveModification(modificationInfos);
+        UUID tabularWith3ModificationUuid = createTabularGeneratorModification(3);
         reset();
-
-        mockMvc.perform(get("/v1/network-modifications/{uuid}", modificationUuid)).andExpectAll(
-                        status().isOk(), content().contentType(MediaType.APPLICATION_JSON))
-                .andReturn();
-        // We check that the request count is not dependent on the number of sub modifications of the tabular modification (the JPA N+1 problem is correctly solved)
+        ApiUtils.getModification(mockMvc, tabularWith3ModificationUuid); // Getting one tabular modification with three sub-modifications
         assertSelectCount(3);
-        reset();
+    }
+
+    @Test
+    public void testSqlRequestsCountOnGetGroupModifications() throws Exception {
+        createTabularGeneratorModification(1);
+        createTabularGeneratorModification(3);
 
-        // We get the modifications of the group (so the 2 tabular modifications)
-        mockMvc.perform(get("/v1/groups/{groupUuid}/network-modifications", getGroupId()))
-                .andExpect(status().isOk());
-        // We check that the request count is not dependent on the number of sub modifications of the tabular modification (the JPA N+1 problem is correctly solved)
+        reset();
+        ApiUtils.getGroupModifications(mockMvc, getGroupId()); // Getting two tabular modifications with respectively one and three sub-modifications
```

Comment:
this call grows in SQL count if we create more tabular modifications
it might not be linked to this PR though, we might have this behaviour with other modifications
Same for other concerned tests